locksmith/src/utils/constants.ts

```diff
@@ -3,3 +3,4 @@ export const baseStripeFee = 30
 export const stripePercentage = 0.029
 export const GAS_COST = 200000 // hardcoded : TODO get better estimate, based on actual execution
 export const GAS_COST_TO_GRANT = 250000
+export const MIN_PAYMENT_STRIPE = 100
```

Comment:
We changed this? @julien51

Reply:
yeah, is the thing i was asking in the prev PR, we have `50` as default